MovieQuiz/Models/MostPopularMovies.swift

```diff
@@ -27,10 +26,11 @@ struct MostPopularMovie: Codable {
         guard let newURL = URL(string: imageUrlString) else {
             return imageURL
         }
+        
         return newURL
     }
     
-    private enum CodingKeys: String, CodingKey {
+    private enum CodingKeys : String, CodingKey {
```

Comment:
Обычно двоеточие так не ставят при объявлении протоколов :) 
```suggestion
    private enum CodingKeys: String, CodingKey {
```